src/main/resources/application.conf

```diff
@@ -33,6 +33,11 @@ docker {
   // dockerToken = ""
 }
 
+jobavoidance {
+  // Set to true in order to avoid running previously run jobs.
+  enabled = false
+}
+
```

Comment:
If the ticket didn't actually specify this it was supposed to say that the default will be off